specification/elastic/resource-manager/readme.md

```diff
@@ -216,6 +225,12 @@ directive:
     where:
       - $.definitions.VMIngestionDetailsResponse.properties.ingestionKey
     reason: Secrets are OK to return in a POST response.
+  - suppress: SECRET_PROPERTY
```

Comment:
Why is this suppression required here?

Reply:
Removed....had added it before due earlier versions.